### DISCUSSION TITLE
rror: (-215:Assertion failed) !ssize.empty() in function 'cv::resize'

### DISCUSSION THREAD

**Discussion by fatiam46** [datasciencecompleteguide/Machine-Learning#21] [General]
**help me please**
import cv2
import numpy as np

####################
widthImg = 640
heightImg = 480
#######################

cap = cv2.VideoCapture(1)
cap.set(3, widthImg)
cap.set(4, heightImg)
cap.set(10, 150)

def preProcessing(img):
    imgGray = cv2.cvtColor(img, cv2.COLOR_BGR2GRAY)
    imgBlur = cv2.GaussianBlur(imgGray, (5, 5), 1)
    imgCanny = cv2.Canny(imgBlur, 200, 200)
    kernel = np.ones((5, 5))
    imgDialte = cv2.dilate(imgCanny, kernel, iterations=2)  # deux fois dialation
    imgThres = cv2.erode(imgDialte, kernel, iterations=1)
    return imgThres

while True:
    success, img = cap.read()
    img = cv2.resize(img, (widthImg, heightImg))
    imgThres = preProcessing(img)
    cv2.imshow("Result", imgThres)
    if cv2.waitKey(1) & 0xFF == ord('q'):
        break
  
![Capture d’écran 2021-08-18 220702](https://user-images.githubusercontent.com/87945994/129972309-aee3543f-b931-4fb5-a2f5-f14232b79a60.png)

**Comment by datasciencecompleteguide** [comment]
Hey,
This error is getting generated because your webcam is not able to read your image or video from your camera.
So, you need to change the interger value in the VideoCapture() function.

Although the link that I am pasting is realted to different error but this will surely help you. Try it out and if then also your problem isn't resolved then I am here for you to help.

[Link to the video](https://youtu.be/hwUQ2_QAdsA)

**Reply by fatiam46** [reply]
thank you so much 
il try it and tell if somthing happend
